src/cts/src/CtsOptions.h

```diff
@@ -148,12 +150,40 @@ class CtsOptions
   {
     clusteringCapacity_ = capacity;
   }
-  double getBufferDistance() const { return bufDistance_; }
-  void setBufferDistance(double distance) { bufDistance_ = distance; }
-  double getVertexBufferDistance() const { return vertexBufDistance_; }
-  void setVertexBufferDistance(double distance)
+
+  // BufferDistance is in DBU
+  int32_t getBufferDistance() const
+  {
+    if (bufDistance_) {
+      return *bufDistance_;
+    }
+
+    if (dbUnits_ == -1) {
+      logger_->error(
+          utl::CTS, 542, "Must provide a dbUnit conversion though setDbUnits.");
+    }
+
+    return 100 /*um*/ * dbUnits_;
+  }
```

Comment:
Out-of-line.  Also why int32_t and not just int?  Similarly  getVertexBufferDistance